.github/labeler.yml

```
@@ -9,3 +9,7 @@ documentation:
 github structure:
   - any: [".github/**/*"]
     all: ["!**/*.py"]
+
+translation:
+  - any: ["custom_components/google_home/translations/*"]
+    all: ["!**/*.py"]
```

I agree with @ArnyminerZ, this should be cleaner
```suggestion
  - any: ["custom_components/google_home/translations/*.json"]
```